nucleus/system/scei/cellos/lv2/sys_process.h

```diff
@@ -22,9 +22,9 @@ enum
     SYS_SPUIMAGE_OBJECT             = 0x22,
     SYS_PRX_OBJECT                  = 0x23,
     SYS_SPUPORT_OBJECT              = 0x24,
-    SYS_UNK65_OBJECT                = 0x41,
-    SYS_UNK66_OBJECT                = 0x42,
-    SYS_UNK67_OBJECT                = 0x43,
+    SYS_SERVICE_LISTENER_OBJECT     = 0x41,  // base system config handle
```

Comment:
As you mentioned, this should be `SYS_CONFIG_OBJECT`.

If you want to commit yourself the update you can do it now. Otherwise I'll pull your changes and apply the fix myself.